current/image-analysis.md

```diff
@@ -0,0 +1,48 @@
+## Image Analysis
+
+AUT supports image analysis, a growing area of interest within web archives.
+
+### Most frequent image URLs in a collection
```

Comment:
This script is so dated - and now that we can just write the images out directly rather than having to wget them, maybe stick with just that?